docs/source/reference/api-server/api-server-tunning.rst

```diff
@@ -0,0 +1,144 @@
+            * Setting the same value for the limits and requests ensures the QoS class of the API server pod being set to ``Guaranteed`` and reduce the chance of the pod being killed by the Kubernetes node when the node is under resource pressure.
+
+            In short, setting the same value for the limits and requests sacrifices the resources utilization for stability and predictability. Pivoting to other trade-off is also possible, but we recommend to keep the memory request and limit the same in production environment to avoid potential eviction caused by Node memory pressure.
+
+    .. tab-item:: VM Deployment
+
+        For VM deployment, in-place vertically scaling the API server is not supported and the API server need to be terminated and recreated to apply the new resources. This means the current state of the API server will be lost. We recommend to create an new API server instance with the new resources and gradually migrate the workload to the new API server.
+
+        Refer to :ref:`sky-api-server-cloud-deploy` for how to deploy the new API server and modify the cluster configuration before running ``sky launch``:
+
+        .. code-block:: diff
+
+            resources:
+            -   cpus: 8+
+            -   memory: 16+
+            +   cpus: 16+
+            +   memory: 32+
+
+The following table shows the maximum concurrency for different resource configurations:
+
+.. list-table::
+   :widths: 1 1 2 2
+   :header-rows: 1
+
+   * - CPU
+     - Memory
+     - Long requests
+     - Short requests
+   * - 4
+     - 8Gi
+     - 8 Max concurrency
+     - 11 Max concurrency
+   * - 16
+     - 32Gi
+     - 32 Max concurrency
+     - 60 Max concurrency
+   * - 32
+     - 64Gi
+     - 64 Max concurrency
+     - 145 Max concurrency
+   * - 64
+     - 128Gi
+     - 128 Max concurrency
+     - 299 Max concurrency
+   * - 128
+     - 256Gi
+     - 256 Max concurrency
+     - 589 Max concurrency
+
+Queuing requests and polling status asynchronously
+--------------------------------------------------
+
+There is no limit on the number of queued requests. So in addition to increasing the allocated resources to improve the maximum concurrency, you can also submit requests with ``--async`` flag and poll the status asynchronously to avoid blocking. For example:
+
+.. code-block:: bash
+
+    # Submit 2000 jobs at once without blocking
+    for i in `seq 1 2000`; do
+        sky jobs launch --async job.yaml
+    done
+    # Poll the status of the jobs
+    sky jobs queue
+
+The requests will be queued on the server and be processed in submission order. If you find the status is not updated for a while, you can inspect the status of the submitted requests with:
+
+.. code-block:: console
+
+    $ sky api status
+    ID                                    User             Name             Created         Status
```

Comment:
Can we reduce the spaces between `User` and `Name` to reduce the width of the table.

Reply:
Sounds good